.github/workflows/sync_labels.yaml

```diff
@@ -2,7 +2,7 @@ name: Sync labels
 
 on:
   pull_request:
-    types: [opened, labeled, unlabeled]
+    types: [opened, edited, labeled, unlabeled]
```

Comment:
This change has no impact if you don't change also `sync_labels.py` 
```

@@ -51,7 +51,7 @@ def get_linked_issues_based_on_pr_body(repo, number):
    pattern = fr'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
    matches = re.findall(pattern, pr.body)
    if not matches:
        raise RuntimeError("No regex matches found in the body!")
        return []
    issue_number_from_pr_body = []
    for match in matches:
        issue_number_from_pr_body.append(match)
        print(f"Found issue number: {match}")
    return issue_number_from_pr_body
def sync_labels(repo, number, label, action, is_issue=False):
    if is_issue:
        linked_prs_or_issues = get_linked_pr_from_issue_number(repo, number)
    else:
        linked_prs_or_issues = get_linked_issues_based_on_pr_body(repo, number)
    for pr_or_issue_number in linked_prs_or_issues:
        if is_issue:
            target = repo.get_issue(pr_or_issue_number)
        else:
            target = repo.get_issue(int(pr_or_issue_number))
        print(pr_or_issue_number)
        if action == 'labeled':
            target.add_to_labels(label)
            print(f"Label '{label}' successfully added.")
        elif action == 'unlabeled':
            target.remove_from_labels(label)
            print(f"Label '{label}' successfully removed.")
        elif action == 'opened':
            copy_labels_from_linked_issues(repo, number)
        else:
            print("Invalid action. Use 'labeled', 'unlabeled' or 'opened'.")
```

Reply:
@tchaikov When thinking about it again, even if you edit the PR and add some `Fixes` to the new issue, you will still need to remove/add the label since we only run during labeled or unlabeled action. We pass the specific label we wish to update